stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -213,6 +213,21 @@ func posixPipe() -> (readFD: CInt, writeFD: CInt) {
   return (fds[0], fds[1])
 }
 
+#if !SWIFT_STDLIB_HAS_ENVIRON
+@_silgen_name("_NSGetEnviron")
+func _NSGetEnviron() -> UnsafeMutablePointer<UnsafeMutablePointer<UnsafeMutablePointer<CChar>?>>
+
+var environ: UnsafeMutablePointer<UnsafeMutablePointer<CChar>?> {
+  #if os(macOS)
+  return _NSGetEnviron().pointee
+  #elseif os(Linux)
+  return __environ
+  #else
+  #error("unsupported platform")
```

Comment:
Please add Windows support.  Windows uses `__environ` for this, but needs to be imported via the module as there is dll storage to be associated with it.

Reply:
Which module? The file already has `import CRT`, is that all that's needed?

Reply:
And I can't really test how this behaves on Windows with SWIFT_STDLIB_HAS_ENVIRON=0... Can I rely on you to sign off on the support for Windows here?

Reply:
Sure, that would be fine.  Yes, `CRT` should provide `environ`.